src/main/java/Jord.java

```diff
@@ -0,0 +1,272 @@
+import tasks.*;
+
+import java.util.Scanner;
+
+public class Jord {
+
+    private static final int TASK_LIMIT = 100;
+    public static final String WELCOME_MESSAGE = "    Hello! I'm Jord\n    What can I do for you?";
+    public static final String BYE_MESSAGE = "    Bye, see you again!";
+
+    private static Task[] TASKS = new Task[TASK_LIMIT];
+    private static int TASK_COUNT = 0;
```

Comment:
Full case variable names should be used with non-mutable variables, perhaps consider the following suggestions 
```suggestion
private static int taskCount = 0;
```